### PR TITLE
Remove note about Kibana versions

### PR DESCRIPTION
**Pull request by dedemorton**
@monicasarbu Not sure if I got the details quite right in this note. Please review.

### CODE REVIEW

File: libbeat/docs/dashboards.asciidoc [hunk 1]
```diff
@@ -6,7 +6,14 @@ the perfect dashboards for your needs. But this flexibility can be a bit
 overwhelming at the beginning, so we have created some sample dashboards to get you
 started and to demonstrate what is possible based on the Beats data.
 
-NOTE: The Beats dashboards require Kibana 4 or later.
+NOTE: The Beats dashboards require Kibana 5 or later.
```
Comment (monicasarbu) [inline libbeat/docs/dashboards.asciidoc]: In fact the Beats dashboards work with both Kibana versions, but Kibana 5.0 works only with Elasticsearch 5.0 (needs to be confirmed by Kibana team). At a second thought, not sure if we should include this information in our dashboards. Maybe we should just avoid this information for now as it's not related to Beats.

Comment (dedemorton) [inline libbeat/docs/dashboards.asciidoc]: @monicasarbu OK, so it sounds like you want me to:
- revert the original statement on line 9 (to say "The Beats dashboards require Kibana 4 or later" )
- remove the note under "Known Issue in 5.0.0-alpha5". I thought there was an issue specific to our dashboards (and that users who have an earlier version of the ES alpha installed would need to move to alpha5 to get our dashboards to work). But it sounds like this is a Kibana + ES issue and should probably be documented in the release notes for those products. 

Is that right?

**Comment by dedemorton** [comment]
@monicasarbu Please review my changes. As we discussed, I removed the note about the Kibana version.

**Comment by monicasarbu** [comment]
Thanks @dedemorton

**Pull request merged**